fs/zonefs_zenfs.cc

```diff
@@ -51,6 +55,42 @@ uint64_t ZoneFsBackend::LBAToZoneOffset(uint64_t pos) {
   return pos - (pos / zone_sz_ * zone_sz_);
 }
 
+std::string ZoneFsBackend::GetBackingDevice(const char *mountpoint) {
+  struct mntent *mnt = NULL;
+  FILE *file = NULL;
+
+  file = setmntent("/proc/mounts", "r");
```

Comment:
Good idea to use `/proc/mounts` instead of `/etc/mtab`. Commit message still says `/etc/mtab`

Reply:
Thanks - I've updated the commit message.